test/unit/test_tables.py

```diff
@@ -1,62 +1,127 @@
 import pytest
 import mock
-from pretty_tables import PrettyTables
-
-
-TABLE = None
-HEADERS = ['ID', 'Name', 'Occupation', 'Employed']
-ROWS = [
-    [1, 'Justin', 'Software Engineer', True],
-    [2, 'Misty', 'Receptionist', False],
-    [3, 'John', None, False],
-]
```

Comment:
Admittedly this was the quick and dirty approach to just get coverage, had every intention of replacing this with a conftest, so thank you!

Reply:
No problem